css-three-column-layout/styles.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+body {
+  color: white;
+  font-family: sans-serif, 'Courier New', Courier;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  background-color: rgb(25, 150, 218);
```

Comment:
I would put the `background-color` property on the `body`, it will fill the entire viewport that way rather than just the contents of the `container` class.